proto/cosmos/gov/v1/gov.proto

```diff
@@ -130,3 +130,31 @@ message TallyParams {
   //  vetoed. Default value: 1/3.
   string veto_threshold = 3 [(cosmos_proto.scalar) = "cosmos.Dec", (gogoproto.jsontag) = "veto_threshold,omitempty"];
 }
+
+// Params defines the parameters for the x/gov module.
+//
+// Since: cosmos-sdk 0.47
+message Params {
+  //  Minimum deposit for a proposal to enter voting period.
+  repeated cosmos.base.v1beta1.Coin min_deposit = 1
+      [(gogoproto.nullable) = false, (gogoproto.jsontag) = "min_deposit,omitempty"];
```

Comment:
Is there anything that changes (e.g. a test failing) if we remove jsontag in this new struct?